azure-pipelines.yml

```diff
@@ -61,833 +52,190 @@ variables:
     value: Products/$(System.TeamProject)/$(Build.Repository.Name)/$(Build.SourceBranchName)/$(Build.BuildNumber)
   - name: Codeql.Enabled
     value: true
-  - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-    - name: _DotNetValidationArtifactsCategory
-      value: .NETCoreValidation
-    - group: DotNet-FSharp-SDLValidation-Params
-  - ${{ if and(eq(variables['System.TeamProject'], 'public'), eq(variables['Build.Reason'], 'PullRequest')) }}:
-    - name: RunningAsPullRequest
-      value: true
-  # Pick up pool provider name behavior from shared yaml template
-  - template: /eng/common/templates/variables/pool-providers.yml
-
-# Variables defined in yml cannot be overridden at queue time; instead overridable variables must be defined in the web UI.
-# Commenting out until something like this is supported: https://github.com/Microsoft/azure-pipelines-yaml/pull/129
-#variables:
-#- name: SkipTests
-#  defaultValue: false
-
-stages:
-- stage: build
-  displayName: Build
-  jobs:
-
-  #-------------------------------------------------------------------------------------------------------------------#
-  #                                                  Signed build                                                     #
-  #-------------------------------------------------------------------------------------------------------------------#
-  - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-    - ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/release/dev17.9') }}:
-      - template: /eng/common/templates/job/onelocbuild.yml
+  - name: _DotNetValidationArtifactsCategory
+    value: .NETCoreValidation
+  - group: DotNet-FSharp-SDLValidation-Params
+  - template: /eng/common/templates-official/variables/pool-providers.yml@self
+
+resources:
+  repositories:
+  - repository: MicroBuildTemplate
+    type: git
+    name: 1ESPipelineTemplates/MicroBuildTemplate
+    ref: refs/tags/release
+
+extends:
+  template: azure-pipelines/MicroBuild.1ES.Official.yml@MicroBuildTemplate
+  parameters:
+    #featureFlags:
```

Comment:
Can be removed?

Reply:
No, I'll leave it there to not forget what to uncomment when baselines will be failing.